src/os/filestore/FileStore.cc

```diff
@@ -107,6 +107,7 @@ using ceph::crypto::SHA1;
 #define XATTR_SPILL_OUT_NAME "user.cephos.spill_out"
 #define XATTR_NO_SPILL_OUT "0"
 #define XATTR_SPILL_OUT "1"
+#define __FUNC__ "function:" << __func__ << ", line:" << __LINE__
```

Comment:
@mychoxin 
I would not make the __FUNC__ string too long. aan somethings like:
`func_so_and_so(12345)` would already be informative.
The other thing you might want to check, is whether `__func__` actually prints the function name, or that is also has the class prefix. (I think it does not have the prefix)